be/src/exec/buffered_reader.cpp

```diff
@@ -116,13 +117,8 @@ Status BufferedReader::_read_once(int64_t position, int64_t nbytes, int64_t* byt
 
 Status BufferedReader::_fill() {
     if (_buffer_offset >= 0) {
-        int64_t bytes_read;
-        // retry for new content
-        int retry_times = 1;
-        do {
-            // fill the buffer
-            RETURN_IF_ERROR(_reader->readat(_buffer_offset, _buffer_size, &bytes_read, _buffer));
-        } while (bytes_read == 0 && retry_times++ < 2);
+        int64_t bytes_read = 0;
+        RETURN_IF_ERROR(_reader->readat(_buffer_offset, _buffer_size, &bytes_read, _buffer));
```

Comment:
Why the previous code had "retry" logic?

Reply:
@xy720 has confirmed, this retry is useless

Reply:
This retry is used to check whether the file has been updated after the first read of end. (nearly useless..)